Lib/test/test_os.py

```diff
@@ -770,7 +770,11 @@ def setUp(self):
         if support.can_symlink():
             os.symlink(os.path.abspath(t2_path), self.link_path)
             os.symlink('broken', broken_link_path, True)
-            self.sub2_tree = (sub2_path, ["link"], ["broken_link", "tmp3"])
+            if os.path.isdir(broken_link_path):
```

Comment:
Okay.  Why is this bugfix part of the "Backport CI config from master" PR?  As far as I can tell, it has nothing to do with backporting the config for AppVeyor, CodeConv, and Travis.

Unless this bugfix is necessary for running AppVeyor, CodeConv, or Travis, I would like this change removed from the PR.